utils/docker/ppc64le.blacklist

```diff
@@ -24,4 +24,3 @@ pmempool_sync_remote
 pmempool_transform_remote
 rpmem_basic
 rpmem_fip
-util_poolset
```

Comment:
> Well, without that it's not possible to tell whether this test passes on ppc64 or not.

Yeah. Here it is. But there are other tests that are already fixed but are still on the blacklist :)